ursa.ino

```diff
@@ -345,16 +345,15 @@ void readMPU6050() {
   rotationX = Wire.read() << 8 | Wire.read();
   rotationY = Wire.read() << 8 | Wire.read();
   rotationZ = Wire.read() << 8 | Wire.read();
-  rotationDPS_X = (rotationX - rotationOffsetX) * 1000000.00 / 32766;//zero gyro with offset values recorded on startup and convert to degrees per second
-  rotationDPS_Y = (rotationY - rotationOffsetY) * 1000000.00 / 32766;
-  rotationDPS_Z = (rotationZ - rotationOffsetZ) * 1000000.00 / 32766;
-  if (micros() > lastCalcedMPU6050) {//try to handle micros' long overflow in a harmless way
-    lastCalcedMPU6050 = micros() - 10;
+  rotationDPS_X = (rotationX - rotationOffsetX) * 1000.00 / 32766;//zero gyro with offset values recorded on startup and convert to degrees per second
+  rotationDPS_Y = (rotationY - rotationOffsetY) * 1000.00 / 32766;
+  rotationDPS_Z = (rotationZ - rotationOffsetZ) * 1000.00 / 32766;
+  if (micros() < lastCalcedMPU6050) {//try to handle micros' long overflow in a harmless way
+    lastCalcedMPU6050 = micros() - 10000;
   }
-  pitch = .99 * ((pitch - pitchOffset) + rotationDPS_X * (micros() - lastCalcedMPU6050) / 1000000.000) + .01 * (degrees(atan2(accelerationY, accelerationZ)) - pitchOffset); //complementary filter combines gyro and accelerometer tilt data in a way that takes advantage of short term accuracy of the gyro and long term accuracy of the accelerometer
+  pitch = .9997 * ((pitch - pitchOffset) + rotationDPS_X * (micros() - lastCalcedMPU6050) / 1000000.000) + .0003 * (degrees(atan2(accelerationY, accelerationZ)) - pitchOffset); //complementary filter combines gyro and accelerometer tilt data in a way that takes advantage of short term accuracy of the gyro and long term accuracy of the accelerometer
```

Comment:
`.9997` is a magic number. Make it a variable?

Reply:
good idea, and .0003 can be replaced with (1-MAGIC_COMPLEMENTARY_FILTER_CONSTANT)